app/build.gradle

```diff
@@ -1,8 +1,9 @@
 apply plugin: 'com.android.application'
+apply from: 'config.gradle'
 
 android {
     compileSdkVersion 21
```

Comment:
IMHO, the compileSDKVersion should be higher than 21, we could use: 
compileSdkVersion 25
 buildToolsVersion '25.0.2'
.......
compile com.android.support:appcompat-v7:25.3.1